.scalafix.conf

```
@@ -0,0 +1,33 @@
+rules = [
+  DisableSyntax
+  LeakingImplicitClassVal
+  NoAutoTupling
+  NoValInForComprehension
+  OrganizeImports
+  ProcedureSyntax
+]
+
+DisableSyntax.noVars = false
+DisableSyntax.noThrows = false
+DisableSyntax.noNulls = false
+DisableSyntax.noReturns = true
+DisableSyntax.noWhileLoops = true
+DisableSyntax.noAsInstanceOf = false
+DisableSyntax.noIsInstanceOf = false
+DisableSyntax.noXml = true
+DisableSyntax.noDefaultArgs = false
+DisableSyntax.noFinalVal = false
+DisableSyntax.noFinalize = true
+DisableSyntax.noValPatterns = false
+DisableSyntax.noUniversalEquality = false
+
+OrganizeImports {
+  coalesceToWildcardImportThreshold = 2147483647 # Int.MaxValue
+  expandRelative = false
+  groupExplicitlyImportedImplicitsSeparately = false
+  groupedImports = Merge
+  groups = ["re:javax?\\.", "scala.", "*"]
```

Can we add a couple groups here ? 

```suggestion
  groups = ["re:javax?\\.", "scala.", "cats.", "weaver.", "*"]
```